### DISCUSSION TITLE
UNRESOLVED_IMPORT 'swiper/element/bundle'

### DISCUSSION THREAD

**Discussion by lucapale** [nolimits4web/swiper#6834] [Q&A]
Hi. I have upgraded my project with Ionic 7 and Swiper 10.
I keep getting this error when I try to register the bundle:
Bundling Warning UNRESOLVED_IMPORT
           'swiper/element/bundle' is imported by .../app-root/app-root.tsx, but could not be resolved
This makes the whole swiper element not working in the app.
The project is based on StencilJS, so it's should be pretty straight forward, but I don't manage to solve the issue.
I also tried downgrading to Swiper 9, but that's the same.
Anybody can help?
Thanks

**Comment by ghost** [comment]
I have the same exact issue and couldn't find any answer or solution for that , do you still have the issue or you managed to solve it ? @lucapale
